Trouble building?

I cloned the project locally, and had issues building:

> fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/main/x86_64/APKINDEX.tar.gz
> ERROR: http://dl-cdn.alpinelinux.org/alpine/v3.6/main: temporary error (try again later)fetch http://dl-cdn.alpinelinux.org/alpine/v3.6/community/x86_64/APKINDEX.tar.gz
> 
> WARNING: Ignoring APKINDEX.84815163.tar.gz: No such file or directory
> ERROR: http://dl-cdn.alpinelinux.org/alpine/v3.6/community: temporary error (try again later)
> WARNING: Ignoring APKINDEX.24d64ab1.tar.gz: No such file or directory
> 

I did some debugging, trying to hack around the build a bit, but could only get so far:

> checking whether DBD::SQLite works... configure: error: in `/nix-1.11.6':
> configure: error: The Perl module WWW::Curl is missing.
> See `config.log' for more details
> 

For some reason I had to make local changes to the Dockerfile to even get that far, even though it looked like it should have worked as-is:

diff --git a/builder/Dockerfile b/builder/Dockerfile
index f76f539..14974ba 100644
--- a/builder/Dockerfile
+++ b/builder/Dockerfile
@@ -9,7 +9,7 @@ RUN echo 'nixbld:x:998:nobody' >> /etc/group && \
         e2fsprogs e2fsprogs-extra && \
     apk add -U perl-www-curl --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ && \
     rm -rf /var/cache/apk/* && \
-    curl -O http://nixos.org/releases/nix/nix-${nversion}/nix-${nversion}.tar.bz2 && \
+    curl -O  https://nixos.org/releases/nix/nix-1.11.6/nix-1.11.6.tar.bz2 && \
     echo "${nsha}  nix-${nversion}.tar.bz2" | sha256sum -c && \
     tar -xjf nix-${nversion}.tar.bz2 && \
     sed -i '1i#include <string.h>' nix-${nversion}/src/libutil/affinity.cc && \


I had also hacked out the proxy settings, since I was trying to build this outside the Oracle network.

--- a/builder/Dockerfile
+++ b/builder/Dockerfile
@@ -3,13 +3,14 @@ FROM alpine
 ENV USER=root
 ARG nversion=1.11.6
 ARG nsha=c3992530762322aa3c23a5d2c6df20a43cb255bc7969c4cd016f6a4262ca843c
+run sed -i -e 's/v3\.6/edge/g' /etc/apk/repositories
+run echo http://dl-cdn.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories
 RUN echo 'nixbld:x:998:nobody' >> /etc/group && \
     apk add -U bash perl libressl bzip2 sqlite curl xz perl-dbd-sqlite perl-dbi libsodium gcc g++ make \
         perl-dev libressl-dev bzip2-dev sqlite-dev curl-dev xz-dev libsodium-dev linux-headers libbz2 libstdc++ \
-        e2fsprogs e2fsprogs-extra && \
-    apk add -U perl-www-curl --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ && \
+        e2fsprogs e2fsprogs-extra perl-www-curl && \
     rm -rf /var/cache/apk/* && \
-    curl -O http://nixos.org/releases/nix/nix-${nversion}/nix-${nversion}.tar.bz2 && \
+    curl -OL https://nixos.org/releases/nix/nix-${nversion}/nix-${nversion}.tar.bz2 && \
     echo "${nsha}  nix-${nversion}.tar.bz2" | sha256sum -c && \
     tar -xjf nix-${nversion}.tar.bz2 && \
     sed -i '1i#include <string.h>' nix-${nversion}/src/libutil/affinity.cc && \

